pages/blog/index.js

```diff
@@ -40,22 +56,51 @@ export default function BlogIndexPage() {
             </p>
             <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">
               Want to publish a blog post? We love community stories.
-              <a className="ml-1 text-primary-500 hover:text-primary-400" href="https://github.com/asyncapi/website/issues/new?template=blog.md" target="_blank" rel="noreferrer">Submit yours!</a>
+              <a
+                className="ml-1 text-primary-500 hover:text-primary-400"
+                href="https://github.com/asyncapi/website/issues/new?template=blog.md"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Submit yours!
+              </a>
             </p>
             <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">
-              We have an <img className="ml-1 text-primary-500 hover:text-primary-400" style={{ display: 'inline' }} src="/img/logos/rss.svg" height="18px" width="18px" /> <a className="ml-1 text-primary-500 hover:text-primary-400" href="/rss.xml">RSS Feed</a> too!
+              We have an{" "}
+              <img
+                className="ml-1 text-primary-500 hover:text-primary-400"
+                style={{ display: "inline" }}
+                src="/img/logos/rss.svg"
+                height="18px"
+                width="18px"
+              />{" "}
+              <a
+                className="ml-1 text-primary-500 hover:text-primary-400"
+                href="/rss.xml"
+              >
+                RSS Feed
+              </a>{" "}
+              too!
             </p>
           </div>
+          <div className="mt-4">
+            <Filter
+              data={navItems}
+              filteredData={posts}
+              onFilter={onFilter}
+              className="w-full inline-flex mx-px justify-center sm:mt-0 sm:w-1/5 sm:text-sm"
```

Comment:
```suggestion
              className="w-full inline-flex mx-px justify-center md:mt-0 md:w-1/5 md:text-sm"
```

I'd break on `md` instead of `sm`. It looks weird if the width is 1/5 on `sm` screen widths.